en/index.md

```diff
@@ -7,8 +7,11 @@ Welcome to the Cocos Creator User Manual! Cocos Creator is a free, cross-platfor
 ## What's New
 
 - Starting with v2.1, Cocos Creator introduced 3D support, 3D features can greatly enrich the performance of 2D games, reduce the resource cost of 2D game. Please refer to [3D Node](3d/index.md).
-- Starting with v2.0.4, Cocos Creator support posted to [Google Play Instant](publish/publish-android-instant.md).
-- Starting with v1.10, Cocos Creator has refactored the underlying resource types, most of which are unaffected, but some projects may receive warnings, please refer to [v1.10 Resource Upgrade Guide](release-notes/raw-asset-migration.md) - Starting with v1.9, Cocos Creator support posted to [Facebook Instant Games](publish/publish-fb-instant-games.md) and [QQ Play](publish/publish-qqplay.md).
+- [v2.1 Release Notes](release-notes/upgrade-guide-v2.1.md)
```

Comment:
`Please refer to [3D Node](3d/index.md)`. -> `Please refer to [v2.1 Release Notes](release-notes/upgrade-guide-v2.1.md)`